src/components/dev-alert-banner/index.tsx

```diff
@@ -0,0 +1,22 @@
+import AlertBanner from 'components/alert-banner'
+
+function DevAlertBanner(): React.ReactElement {
```

Comment:
Added `DevAlertBanner` to decouple the chained `docs-layout` >> `base-new-layout` while not repeating this code in both layouts.